packages/core/src/components/VerticalNavigation/Navigation/Navigation.tsx

```diff
@@ -357,6 +359,7 @@ export const HvVerticalNavigationTree = ({
               data={navigationPopup.data}
               onClose={handleNavigationPopupClose}
               onChange={handleChange}
+              popupStyles={popupStyles}
```

Comment:
Is overriding inline `styles` really necessary for your use case? If the goal is styling, then we should use classes instead of inline styles. Classes allow for much more customisation, and is our recommended approach for styling components.

I'm referring to something like this:
```suggestion
              className={classes?.popup}
```